### PR TITLE
[locale] relativeTime week transaltion for most commonly used languages

### PR DESCRIPTION
**Pull request by jguc**
Added missing locale for relativeTime weeks which were added in moment 2.25
Not all languages are covered but tried to add the most popular ones and mainly those that I can verify with high certainty. No base tests exist for any of the relativeTime format so no new tests have been created.

Added week support for :
pl, bg, es, fr, it, nb, nn, nl, pt, ro, ru, th, tr, vi, zh-cn 

- [x] pl - native, no need to confirm 
- [x] bg - @kraz
- [x] es - @julionc
- [x] fr - @jfroffice
- [x] it - @aliem / @nostalgiaz / @Manfre98
- [x] nb - @rexxars / @sigurdga / @stephenramthun
- [x] nn - @sigurdga
- [x] nl - @jorisroling / @middagj
- [x] pt - @jalex79
- [x] ro - @gurdiga / @avaly / @cepem
- [x] ru - @Viktorminator / @Oire / @socketpair
- [x] th - @sirn
- [x] tr - @erhangundogan / @erhangundogan
- [x] vi - @bangnk / @chienkira
- [x] zh-cn - @suupic / @zenozeng / @uu109

### CODE REVIEW

File: src/locale/ru.js [hunk 1]
```diff
@@ -20,6 +20,7 @@ function relativeTimeWithPlural(number, withoutSuffix, key) {
         mm: withoutSuffix ? 'минута_минуты_минут' : 'минуту_минуты_минут',
         hh: 'час_часа_часов',
         dd: 'день_дня_дней',
+        ww: 'неделя_недели_недель',
```
Comment (socketpair) [inline src/locale/ru.js:23]: OK

File: src/locale/ru.js [hunk 2]
```diff
@@ -160,6 +161,8 @@ export default moment.defineLocale('ru', {
         hh: relativeTimeWithPlural,
         d: 'день',
         dd: relativeTimeWithPlural,
+        w: 'неделя',
```
Comment (socketpair) [inline src/locale/ru.js:164]: OK

**Review by sirn — APPROVED** [review]
th looks fine for me.

File: src/locale/tr.js [hunk 3]
```diff
@@ -74,6 +74,8 @@ export default moment.defineLocale('tr', {
         hh: '%d saat',
         d: 'bir gün',
         dd: '%d gün',
+        w: 'bir hafta',
+        ww: '%d hafta',
```
Comment (erhangundogan) [inline src/locale/tr.js:78]: Looks good to me 👍

File: src/locale/fr.js [hunk 4]
```diff
@@ -67,6 +67,8 @@ export default moment.defineLocale('fr', {
         hh: '%d heures',
         d: 'un jour',
         dd: '%d jours',
+        w: 'un semaine',
```
Comment (jfroffice) [inline src/locale/fr.js:70]: need to be `une semaine`

File: src/locale/nl.js [hunk 5]
```diff
@@ -83,6 +83,8 @@ export default moment.defineLocale('nl', {
         hh: '%d uur',
         d: 'één dag',
         dd: '%d dagen',
+        w: 'één week',
```
Comment (middagj) [inline src/locale/nl.js:86]: 👍

**Review by middagj — APPROVED** [review]

File: src/locale/ro.js [hunk 6]
```diff
@@ -12,6 +12,7 @@ function relativeTimeWithPlural(number, withoutSuffix, key) {
             mm: 'minute',
             hh: 'ore',
             dd: 'zile',
+            ww: 'saptamani',
             MM: 'luni',
```
Comment (cepem) [inline src/locale/ro.js:16]: Should be "săptămâni" instead, in order to stay consistent with the diacritics.

**Review by cepem — CHANGES_REQUESTED** [review]

**Review by cepem — APPROVED** [review]
LGTM!

**Review by erhangundogan — APPROVED** [review]

File: src/locale/bg.js [hunk 7]
```diff
@@ -53,6 +53,8 @@ export default moment.defineLocale('bg', {
         hh: '%d часа',
         d: 'ден',
         dd: '%d дена',
+        w: 'седмица',
+        ww: '%d седмици',
```
Comment (kraz) [inline src/locale/bg.js:57]: LGTM :thumbsup: :rocket:

File: src/locale/es.js [hunk 8]
```diff
@@ -89,6 +89,8 @@ export default moment.defineLocale('es', {
         hh: '%d horas',
         d: 'un día',
         dd: '%d días',
+        w: 'un semana',
```
Comment (julionc) [inline src/locale/es.js:92]: Should be `una semana` instead.

**Review by julionc — CHANGES_REQUESTED** [review]

File: src/locale/nb.js [hunk 9]
```diff
@@ -45,6 +45,8 @@ export default moment.defineLocale('nb', {
         hh: '%d timer',
         d: 'en dag',
         dd: '%d dager',
+        w: 'en uke',
+        ww: '%d uker',
```
Comment (sigurdga) [inline src/locale/nb.js:49]: Looks good! If you want to add nynorsk (nn) as well, it is 'ei veke' / '%d veker'

**Review by sigurdga — APPROVED** [review]

File: src/locale/nb.js (hunk 9 above)
Comment (jguc) [reply src/locale/nb.js:49]: Thanks @sigurdga ! nn added !

**Review by uu109 — APPROVED** [review]
looks good to me

File: src/locale/zh-cn.js [hunk 10]
```diff
@@ -105,6 +105,8 @@ export default moment.defineLocale('zh-cn', {
         hh: '%d 小时',
         d: '1 天',
         dd: '%d 天',
+        w: '1 周',
+        ww: '%d 周',
```
Comment (uu109) [inline src/locale/zh-cn.js:109]: ok

**Review by uu109 — APPROVED** [review]

**Comment by marwahaha** [comment]
@jguc - Thanks a lot for this!
If you'd like, you can move all unapproved changes to a separate PR and I can merge the approved changes now.

**Comment by jguc** [comment]
> @jguc - Thanks a lot for this!
> If you'd like, you can move all unapproved changes to a separate PR and I can merge the approved changes now.

@marwahaha don't mention, the pleasure is all mine.
That's what I thought initially, to split the PR with the remaining translations, just give me one more day to ask some more people for verification and see how that goes.

File: src/locale/it.js [hunk 11]
```diff
@@ -90,6 +90,8 @@ export default moment.defineLocale('it', {
         hh: '%d ore',
         d: 'un giorno',
         dd: '%d giorni',
+        w: 'una settimana',
+        ww: '%d settimane',
```
Comment (jguc) [inline src/locale/it.js:94]: Hi @afilini or @atorin can I ask you guys to have a look at two words in Italian in this PR to moment.js library and provide feedback. (I found you at Developers Italia repo, sorry for bothering you but I thought to give it a go)

File: src/locale/pt.js [hunk 12]
```diff
@@ -46,6 +46,8 @@ export default moment.defineLocale('pt', {
         hh: '%d horas',
         d: 'um dia',
         dd: '%d dias',
+        w: 'uma semana',
+        ww: '%d semanas',
```
Comment (jguc) [inline src/locale/pt.js:50]: Hi @caio-ribeiro-pereira, can you verify those two lines?

File: src/locale/vi.js [hunk 13]
```diff
@@ -61,6 +61,8 @@ export default moment.defineLocale('vi', {
         hh: '%d giờ',
         d: 'một ngày',
         dd: '%d ngày',
+        w: 'một tuần',
+        ww: '%d tuần',
```
Comment (jguc) [inline src/locale/vi.js:65]: Hi @1ec5, are those two lines something you can verify?

**Comment by jguc** [comment]
Hi, @judikdavid / @bankyadam please verify the Hungarian version.

File: src/locale/it.js (hunk 11 above)
Comment (jguc) [reply src/locale/it.js:94]: + @zenlor maybe you can verify?

File: src/locale/vi.js (hunk 13 above)
Comment (1ec5) [reply src/locale/vi.js:65]: Yes, this is correct.

**Review by 1ec5 — APPROVED** [review]
Vietnamese looks good to me.

File: src/locale/pt.js (hunk 12 above)
Comment (caio-ribeiro-pereira) [reply src/locale/pt.js:50]: This is okay

File: src/locale/it.js (hunk 11 above)
Comment (zenlor) [reply src/locale/it.js:94]: LGTM

Comment (atorin) [reply src/locale/it.js:94]: Yes it's fine, nothing to complain about!

**Comment by jguc** [comment]
@marwahaha ready to merge, I've only removed HU due to lack of verification

**Comment by marwahaha** [comment]
Thanks again!

**Pull request merged**